beets/library.py

```diff
@@ -259,7 +259,7 @@ class LibModel(dbcore.Model):
     """Shared concrete functionality for Items and Albums.
     """
 
-    _format_config_key = None
+    format_config_key = None
```

Comment:
Aha, I get it. Good point. I think I still prefer the underscore to suggest an "internal detail" even if it's manipulated from outside in a couple of points.